packages/@okta/i18n/src/properties/login.properties

```diff
@@ -678,6 +679,7 @@ signin.with.fastpass = Signing in using Okta FastPass
 customUri.title = Verify account access
 customUri.subtitle = Launching Okta Verify...
 customUri.content = <div class="skinny-content"> If nothing prompts from the browser, <a href="#" id="launch-ov" class="link">click here</a> to launch Okta Verify, or make sure Okta Verify is installed.</div>
+customUri.required.content = <div class="skinny-content">Please click&nbsp;<span class="strong">Open oktaverify.app</span>&nbsp;if you see the system dialog.</div><div class="skinny-content">If nothing prompts from the browser,&nbsp;<a href="#" id="launch-ov" class="link">click here</a>&nbsp;to launch Okta Verify, or&nbsp;<a href="{0}" target="_blank" id="download-ov" class="link">download & run Okta Verify</a>.</div>
```

Comment:
I have never seen this much of html inside a properties file. I don' think it's ideal. We should just keep the text and move out html to js file.